README.md

```diff
@@ -1 +1,49 @@
-# java-jaxrs
\ No newline at end of file
+# OpenTracing JAX-RS Instrumentation
+
+Warning: This library is still a work in progress!
+
+[![Travis](https://travis-ci.org/opentracing-contrib/java-jaxrs.svg?branch=master)](https://travis-ci.org/opentracing-contrib/java-jaxrs)
+
+OpenTracing instrumentation for JAX-RS standard. 
+It supports server and client request tracing.
+
+## Tracing Server Requests
+```
+// register this in javax.ws.rs.core.Application
+ServerTracingDynamicFeature.Builder
+    .traceAll(yourPreferredTracer)
+    .withStandardTags()
```

Comment:
I prefer to expose how to install custom tags vs a flag like this. in other words, you either use defaults or you compose something on top of them. This customization has been requested many times from different people in Brave, and will be available shortly. (customization can be a later hook, btw)